exporter/azure_monitor/azure_monitor.go

```diff
@@ -63,14 +63,14 @@ func (exporter *AzureTraceExporter) ExportSpan(sd *trace.SpanData) {
 			Success : true,
 		}
 		if _, isIncluded := sd.Attributes["http.method"]; isIncluded {
-			currentData.Name = sd.Attributes["http.method"].(string)
+			currentData.Name = fmt.Sprintf("%s", sd.Attributes["http.method"])
 		}
 		if _, isIncluded := sd.Attributes["http.url"]; isIncluded {
-			currentData.Name = currentData.Name + " " + sd.Attributes["http.url"].(string)
-			currentData.Url = sd.Attributes["'http.url"].(string)
+			currentData.Name = currentData.Name + " " + fmt.Sprintf("%s", sd.Attributes["http.url"])

```

Comment:
This probably should be `fmt.Sprintf("%s %s", currentData.Name, sd.Attributes["http.url"])`.

Reply:
I agree. This is cleaner. Will make change soon.